pkg/prometheus/operator_test.go

```diff
@@ -394,3 +394,73 @@ func TestValidateRelabelConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateProberUrl(t *testing.T) {
+	for _, tc := range []struct {
+		proberSpec     monitoringv1.ProberSpec
+		validProberURL bool
+	}{
+		{
+			proberSpec: monitoringv1.ProberSpec{
```

Comment:
Can you add a name to the test cases to indicate what each test case covers?  You can then use the name in `t.Run`.